pages/org/urbanscape/index.jsx

```diff
@@ -61,27 +68,61 @@ const UrbanScapePage = () => {
         toast.info('Form submitted, should approve application');
         setIsSuccess(true);
     }
+
+    const { refetch } = useQrCode({ proofTemplateId });
+
+    useEffect(() => {
+        refetch();
+        // eslint-disable-next-line
+    }, []);
+
+    useEffect(() => {
+        if (verified === true) {
+            setTimeout(() => {
```

Comment:
I don't understand this `setTimeout`. This `useEffect` has `verified, retrievedData` as deps, which will run when either change. 

Maybe there is something I'm missing here, but why don't you just check `if (verified === true && receivedData !== null)`

Reply:
Thats right! Changed to :
```javascript
if (verified === true && retrievedData !== null) {
```